yarn-project/aztec-rpc/src/kernel_prover/kernel_prover.ts

```diff
@@ -264,4 +276,27 @@ export class KernelProver {
     }
     return hints;
   }
+
+  private getNullifierHints(
+    nullifiedCommitments: Tuple<Fr, typeof MAX_NEW_NULLIFIERS_PER_TX>,
+    commitments: Tuple<Fr, typeof MAX_NEW_COMMITMENTS_PER_TX>,
+  ): Tuple<Fr, typeof MAX_NEW_NULLIFIERS_PER_TX> {
```

Comment:
Comment on this function would be helpful

Reply:
I have added comments on both getReadRequestHints and getNullifierHints.